apps/full-site-editing/full-site-editing-plugin/starter-page-templates/page-template-modal/index.js

```diff
@@ -136,7 +136,16 @@ class PageTemplateModal extends Component {
 		// Only overwrite the page title if the template is not one of the Homepage Layouts
 		const title = isHomepageTemplate ? null : this.getTitleByTemplateSlug( slug );
 
-		// Skip inserting if there's nothing to insert.
+		// Check to see if this is a blank template selection
+		// and reset the template if so.
+		if ( 'blank' === slug ) {
+			this.props.insertTemplate( '', [] );
```

Comment:
Just noting that this leaves the page title to whatever is currently set - it doesn't clear it. I _think_ that's fine.

Reply:
I originally cleared the title, but it felt wrong, especially if the user has already added a custom title of their own.